app/models/user.rb

```diff
@@ -13,6 +15,7 @@ class User < ActiveRecord::Base
     after_initialize do
         self.secret ||= SecureRandom.base64(24)
         self.username ||= UsernameGenerator.new
+        self.avatar ||= Identicon.data_url_for self.secret, 256, [255, 255, 255]
```

Comment:
The honest reason is actually pretty dumb. Basically I wanted the avatars to look different between DB resets so I could see different randomly generated avatars more easily.